File: flut/.flutter/lib/flutter/widgets/scroll_configuration.dart
import 'package:flutter/material.dart';
import 'package:flut/flut/runtime.dart';
import 'package:flut/flut/object.dart';

class FlutScrollBehavior extends FlutValueObject {
  final ScrollBehavior value;
  const FlutScrollBehavior(this.value) : super('ScrollBehavior');

  @override
  Map<String, dynamic> flutEncode() {
    final result = flutBaseProps();
    return result;
  }

  static ScrollBehavior? flutDecode(
    FlutRuntime runtime,
    Map<String, dynamic> data,
  ) {
    return const ScrollBehavior();
  }
}

File: flut/flutter/widgets/scroll_configuration.py
from typing import override

from flut._flut.engine import FlutValueObject


class ScrollBehavior(FlutValueObject):
    _flut_type = "ScrollBehavior"

    def __init__(self):
        super().__init__()

    @override
    def _flut_pack(self) -> dict:
        result = self._flut_base_props()
        return result

    @staticmethod
    def _flut_unpack(data: dict) -> "ScrollBehavior":
        return ScrollBehavior()
